### DISCUSSION TITLE
教員証明がリジェクトされました

### DISCUSSION THREAD

**Discussion by ghost** [community/community#53218] [GitHub Education]
教員証明を申請したところ、次のようなメールが届きました：
You are unlikely to be verified until you have completed your [GitHub user profile](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile) with your full name as it appears in your academic affiliation document plus a short bio.

教員証明の受諾に必要な具体的なアクションをご教示ください。

**Comment by samus-aran** [comment]
Hi @EnchanterWug , thanks for participating!
We take our [Code of Conduct](https://github.com/community/community/blob/0ca37cf594e438fd4aec3c7f3736b95e6b0b3567/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
